src/components/ProductListItem.vue

```diff
@@ -6,17 +6,15 @@
     <ion-label class="ion-text-wrap">
       <h2>{{ getProductIdentificationValue(productIdentificationPref.primaryId, getProduct(item.productId)) }}</h2>
       <p class="ion-text-wrap">{{ getProductIdentificationValue(productIdentificationPref.secondaryId, getProduct(item.productId)) }}</p>
-      <p>Color: color</p>
-      <p>Size: size</p>
     </ion-label>
     <!-- Only show stock if its not a ship to store order -->
     <div v-if="!isShipToStoreOrder">
       <ion-button v-if="!isFetchingStock && !showInfoIcon" fill="clear" @click.stop="fetchProductStock(item.productId)">
         <ion-icon color="medium" slot="icon-only" :icon="cubeOutline" />
       </ion-button>
       <div v-else-if="showInfoIcon" class="atp-info">
-        <ion-note slot="end"> 50 ATP </ion-note>
-        <ion-button fill="clear" @click.stop="getInventoryComputationDetails($event)">
+        <ion-note slot="end"> {{ getInventoryInformation(item.productId).onlineAtp ?? '0' }} </ion-note>
```

Comment:
We can update the checks here, to first check whether the onlineAtp is available or not, if available display the atp and infoIcon, if not check the value for isFetchingStock field, and at last display the cube icon.

Also I think there is no need for having showInfoIcon variable, we can handle the scenario directly in the above case.